dev/com.ibm.ws.microprofile.reactive.messaging_fat/publish/servers/ProducerRecordMessagingServer/bootstrap.properties

```diff
@@ -0,0 +1,16 @@
+###############################################################################
+# Copyright (c) 2017, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+osgi.console=5471
+com.ibm.ws.logging.trace.specification = *=info:logservice=detail:REACTIVEMESSAGE=all:com.ibm.ws.microprofile.reactive.messaging.kafka.adapter.impl.*=FINEST
+
+# This is an override to force the new beta-level code to be used in these FAT tests. It should be removed before GA.
+use.kafka.producer.record=true
```

Comment:
Silly question: have you run the tests with this property set on the other test servers to check it doesn't cause any regressions?

Reply:
Not a silly question ... I think I might change the tests so that they're not using a separate server after all.